geode-core/src/main/java/org/apache/geode/internal/cache/versions/RegionVersionHolder.java

```diff
@@ -280,63 +292,37 @@ private synchronized void mergeBitSet() {
    * adjusted and a new bitSetVersion is established.
    *
    * @param newVersion the desired new bitSetVersion, which may be > the max representable in the
-   *        bitset
-   * @param numBits the desired number of bits to flush from the bitset
+   *        bitset. This should *always* be a version that has been received, because this
+   *        method may need to create an exception up to this version, and the existance of an
+   *        exception implies that the final version was received.
+   *
+   *
    */
-  private void addBitSetExceptions(int numBits, long newVersion) {
-    final boolean isDebugEnabled_RVV = logger.isTraceEnabled(LogMarker.RVV_VERBOSE);
-    int lastSetIndex = -1;
+  private void addBitSetExceptions(long newVersion) {
 
-    if (isDebugEnabled_RVV) {
-      logger.trace(LogMarker.RVV_VERBOSE, "addBitSetExceptions({},{})", numBits, newVersion);
+    if (newVersion <= bitSetVersion) {
+      return;
```

Comment:
shouldn't we throw an exception instead of ignoring the version?

Reply:
I think the contract for this method is that it adds exceptions out of the bit set up to newVersion. So maybe it's reasonable that it just does nothing if newVersion < bitSetVersion? The old code also just did nothing in that case.

Reply:
It looks like the old method didn't even use the newVersion parameter outside of the trace-level log statement.  It only paid attention to the numBits parameter.